src/Elastic.Documentation.Site/Htmx.cs

```diff
@@ -24,8 +31,9 @@ public static string GetHxAttributes(
 		string? hxIndicator = HxIndicator
 	)
 	{
+		var hxGetUrl = targetUrl.Contains('?') ? $"{targetUrl}&v={VersionHash}" : $"{targetUrl}?v={VersionHash}";
```

Comment:
It seems like the targetUrl may already contain a fragment and this might created a malformed url.